src/app/quizPage.js

```diff
@@ -0,0 +1,269 @@
+import {
```

Comment:
Dobrze byłoby to rozbić na pliki dla poszczególnym komponentów widoku. Ciężko się odnaleźć w tak długich plikach.